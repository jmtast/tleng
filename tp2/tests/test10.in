// Error de sem√°ntica: Instrumento mayor a 127
#tempo negra 120
#compas 2/4

const oct1 = 5;

voz (128)
{
  compas
  {
    silencio(blanca);
  }
}
