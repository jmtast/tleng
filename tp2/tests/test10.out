Error:
Instrumento inválido. Debe estar entre 0 y 127. Línea 7
